domain/tshirt.js

```diff
@@ -10,19 +10,26 @@ function invalidData(body) {
   return _.isUndefined(body.size) || _.isUndefined(body.model);
 }
 
+tshirt.NoTShirt = function () {
+  return {
+    details: [{ size: "L", model: "Slim fit" }],
```

Comment:
And does it make sense, to provide details if we know that, no tshirt has been ordered?